data/strings/strings.txt

```diff
@@ -20153,93 +20153,95 @@
-    zh-Hans = 添加起点以规划路线
-    zh-Hant = 新增起點以計劃路線
+    comment = User selected the destination by pressing Route To, but the current position is unknown. User needs to select a starting point of a route using search or by tapping on the map and then pressing "Route From".
+    en = Use search or tap on the map to add a route starting point
+    af = Gebruik soek of tik op die kaart om 'n roetebeginpunt by te voeg
+    ar = استخدام البحث أو النقر على الخريطة لإضافة نقطة بداية الطريق
+    az = Marşrutun başlanğıc nöqtəsi əlavə etmək üçün axtarışdan istifadə edin və ya xəritədə klikləyin
+    be = Скарыстайцеся пошукам або націсніце на мапе, каб дадаць пачатак маршруту
+    bg = Използвайте търсене или докоснете картата, за да добавите начална точка на маршрута
+    ca = Utilitzeu la cerca o toqueu el mapa per afegir un punt de partida de la ruta
+    cs = Pomocí vyhledávání nebo klepnutím na mapu přidejte výchozí bod trasy
+    da = Brug søgning eller tryk på kortet for at tilføje et startpunkt for en rute
+    de = Verwende die Suche oder tippe auf die Karte, um einen Routenstartpunkt hinzuzufügen
+    el = Χρησιμοποιήστε την αναζήτηση ή πατήστε στο χάρτη για να προσθέσετε ένα σημείο εκκίνησης της διαδρομής
+    es = Utiliza la búsqueda o toca en el mapa para añadir un punto de inicio de ruta
+    et = Marsruudi alguspunkti lisamiseks kasutage otsingut või puudutage kaarti
+    eu = Erabili bilaketa edo sakatu mapan ibilbidearen abiapuntua gehitzeko
+    fa = برای افزودن نقطه شروع مسیر، از جستجو استفاده کنید یا روی نقشه ضربه بزنید
+    fi = Käytä hakua tai napauta karttaa lisätäksesi reitin lähtöpisteen
+    fr = Utilise la recherche ou tape sur la carte pour ajouter un point de départ d'itinéraire
+    he = השתמש בחיפוש או הקש על המפה כדי להוסיף נקודת התחלה למסלול
+    hi = मार्ग का प्रारंभिक बिंदु जोड़ने के लिए खोज का उपयोग करें या मानचित्र पर टैप करें
+    hu = Használja a keresést, vagy koppintson a térképre az útvonal kezdőpontjának hozzáadásához
+    id = Gunakan pencarian atau ketuk pada peta untuk menambahkan titik awal rute
+    it = Usa la ricerca o tocca la mappa per aggiungere un punto di partenza del percorso
+    ja = 検索を使うか、地図上をタップしてルートの出発点を追加する
+    ko = 검색을 사용하거나 지도를 탭하여 경로 시작점을 추가합니다
+    lt = Naudokite paiešką arba bakstelėkite žemėlapyje, kad pridėtumėte maršruto pradžios tašką
+    mr = मार्ग प्रारंभ बिंदू जोडण्यासाठी नकाशावर शोध वापरा किंवा टॅप करा
+    nb = Bruk søk eller trykk på kartet for å legge til et startpunkt for ruten
+    nl = Gebruik zoeken of tik op de kaart om een startpunt voor de route toe te voegen
+    pl = Użyj funkcji wyszukiwania lub dotknij mapy, aby dodać punkt początkowy trasy
+    pt = Utiliza a pesquisa ou toca no mapa para adicionar um ponto de partida do percurso
+    pt-BR = Use a busca ou toque no mapa para adicionar um ponto de partida da rota
+    ro = Utilizați funcția de căutare sau atingeți ușor pe hartă pentru a adăuga un punct de plecare al traseului
```

Comment:
You can leave off the "al traseului" at the end, to make it symmetrical with the one below.